packages/lsp-compliant-services/src/index.ts

```diff
@@ -64,7 +64,7 @@ export * from './handlers/CodeLensHandler';
 // Export services
 export * from './services/DocumentProcessingService';
 export * from './services/DocumentSaveProcessingService';
-export * from './services/ParseResultCache';
+export * from './services/DocumentStateCache';
```

Comment:
I've been trying to get rid of `export *` and be a bit more surgical about what needs to be exported.  I've also caught these hiding unused stuff.

I've been trying to get rid of barrels entirely (import what you need from where it lives)